src/main/resources/Documentation/about.md

```diff
@@ -8,11 +8,44 @@ Parameters
 
 Project Configuration:
 
-TODO: Add Configuration parameters and instructions.
+eiffel-agen.enabled
+:   Plugin state. if set to `true`, send message Eiffel message for every commit submitted for this repository.
+    Plugin considered disabled if this option is not defined or set to `false`.
+
+eiffel-agen.filter
+:   Allow users to defined a white list of branches, i.e. messages will be sent only for a commits submitted on specified branches, all other submits will be ignored.
+    Accepts Java regex's separated by spaces. If not defined, send messages for commits submitted on all branches.
+
+eiffel-agen.flow-context
+:   Optional, but expects a UUID of a EiffelFlowContextDefinedEvent if provided.
```

Comment:
It can be multiple events, they will link to the FLOW-CONTEXT link type.
In a CI flow you should be able to know which FLOW this events belong to. It can be a single or multiple Flows.

Reply:
For the first iteration, we are expecting to provide the UUID, but later it will support lookups which will try to fetch the UUID and add them in the events.

Reply:
Good, now I think it's more clear and the link to the Eiffel protocol repository helps guide users who want more information.